### ISSUE TITLE
vscode shows error despite @ts-nocheck 

### ISSUE THREAD

**Issue by cgolinski** [microsoft/vscode-eslint#1363]
Hi vscode-eslint team! Hoping you can help me figure out what's going on here. Thanks!

## Summary

In my `.eslintrc.json` I have the following rule set:
```
"@typescript-eslint/ban-ts-comment": [2, { "ts-nocheck": "allow-with-description" }],
 ```
 
<img width="915" alt="eslintrc json" src="https://user-images.githubusercontent.com/22013506/139830608-53d8078a-0e0d-4074-8b80-4f1ba70348dc.png">
 
When I run eslint (`eslint src/components/TestNoCheckRule/index.tsx`) on the command line, there are no errors or warnings.

<img width="593" alt="command line" src="https://user-images.githubusercontent.com/22013506/139830640-eb4f756d-e98e-44b7-8441-086cb4b5f94e.png">

 
On the first line of the file itself, I have:
```
//@ts-nocheck: this is a test description
```
And yet, in that file, vscode shows a type error: 

<img width="511" alt="ts error in file" src="https://user-images.githubusercontent.com/22013506/139830669-4844b29c-a635-48af-9e3a-a03414829936.png">

## Expected:
vscode should respect `//@ts-nocheck` and not show type errors

## Actual: 
Despite `//@ts-nocheck`, vscode does show type error

**Comment by cgolinski** [comment]
Hmm. In other files, it seems that `//ts-nocheck` isn't being respected by eslint either. So might not be an issue with vscode-eslint. Will investigate further and report back.

**Comment by cgolinski** [comment]
`// @ts-ignore` is being respected by both eslint and vscode-eslint. Not sure why, but `// @ts-nocheck` is not being respected by either. Closing as this doesn't seem to be an issue wtih vscode-eslint.
